### PR TITLE
docs: ip-adapter

### PR DESCRIPTION
**Pull request by efredin**

### CODE REVIEW

File: fern/apis/image-gen/openapi/openapi-overrides.yml [hunk 1]
```diff
@@ -44,6 +44,13 @@ paths:
       servers:
         - url: https://image.octoai.run
           x-name: ImageGen
+  /generate/ip-adapter-sdxl:
+    post:
+      summary: "IP Adapter"
+      x-fern-sdk-method-name: ip_adapter
+      servers:
+        - url: https://image.octoai.run
+          x-name: ImageGen
```
Comment (jpwallace22) [inline fern/apis/image-gen/openapi/openapi-overrides.yml:53]: I don't think we need this anymore. I had them put the 'servers' in the specs

**Review by jpwallace22 — APPROVED** [review]

File: fern/apis/image-gen/openapi/openapi-overrides.yml (hunk 1 above)
Comment (efredin) [reply fern/apis/image-gen/openapi/openapi-overrides.yml:53]: I never had any luck with root level `servers` declarations in the overrides, but I can try it and see if it works.

Comment (efredin) [reply fern/apis/image-gen/openapi/openapi-overrides.yml:53]: I'm going to create a separate PR to attempt removing all the `servers` so we don't keep copy/pasting.

Comment (efredin) [reply fern/apis/image-gen/openapi/openapi-overrides.yml:53]: Doesn't work https://github.com/octoml/fern-config/pull/78.  May need to ask fern why.

**Pull request merged**